bluzelle/chain.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "bluzelle",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://bluzelle.com/",
+  "pretty_name": "Bluzelle",
+  "chain_id": "bluzelle-8",
+  "bech32_prefix": "bluzelle",
```

Comment:
Please register this bech32 prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Actually, it was registered about 12 hrs ago. 

https://github.com/satoshilabs/slips/commit/2457b96b3fde88066de82810ac86aec9370ad1ea